docs/display.rst

```diff
@@ -0,0 +1,69 @@
+Display
+*******
+
+.. py:module:: microbit.display
+
+This module controls the 5×5 LED display on the front of your board. It can
+be used to display images, animations and even text.
+
+Functions
+=========
+
+.. py:function:: get_pixel(x, y)
+
+    Return the brightness of the LED at column ``x`` and row ``y`` as an
+    integer between 0 (dark) and 9 (bright).
+
+
+.. py:function:: set_pixel(x, y, value)
+
+    Set the brightness of the LED at column ``x`` and row ``y`` to ``value``,
+    which has to be an integer between 0 and 9.
+
+
+.. py:function:: clear()
+
+    Set the brightness of all LEDs to 0.
+
+
+.. py:function:: print(string, delay=400)
+
+    Display the ``string`` one letter at a time, with ``delay`` milliseconds
+    between each pair of letters.
```

Comment:
Not related to the documentation itself, just a comment on the choice of name. I think this is a horrible idea. You are basically setting yourself up for people confusing this with the build-in print. Not to mention that "printing" an image doesn't really sound like a good idea.

Possible other names: ``flash``, ``show``, ``write``.

By the way, if I was designing a general API for this, I would make this function check for a magical attribute, similar to ``__repr__``, that would return an iterable of images. Then you could have user objects that can also be displayed, and instead of "animated" function below, you would just have an "Animation" iterable.